@kiva/kv-tokens/configs/tailwind.config.cjs

```diff
@@ -46,6 +46,8 @@ module.exports = {
 			marigold: colors.marigold,
 			'desert-rose': colors['desert-rose'],
 			stone: colors.stone,
+			gray: colors.gray,
```

Comment:
I think this will open up several "grays" that were previously relegated to various other tw classes such as border.

Reply:
Exciting!